prql-compiler/tests/integration/snapshots/integration__fmt@genre_counts.prql.snap

```diff
@@ -1,10 +1,10 @@
 ---
 source: prql-compiler/tests/integration/main.rs
-expression: "let genre_count = (\n    from genres\n    aggregate {a = count name}\n)\n\nfrom genre_count\nfilter a > 0\nselect a = -a\n"
+expression: "# clickhouse:skip (ClickHouse prefers aliases to column names https://github.com/PRQL/prql/issues/2827)\n# mssql:test\nlet genre_count = (\n    from genres\n    aggregate {a = count name}\n)\n\nfrom genre_count\nfilter a > 0\nselect a = -a\n"
 input_file: prql-compiler/tests/integration/queries/genre_counts.prql
 ---
-let genre_count = (from genres | aggregate {a = count name})
+let genre_count = (from genres | aggregate {a = (count name)})
```

Comment:
These parenthesis are not needed, idiomatic PRQL should not have them. This can be improved in a latter PR.

Reply:
This is the issue between whether an assign is in a tuple or not — do we want different canonical styles based on that?